frontend/src/components/CytoscapeGraph/MiniGraphCard.tsx

```diff
@@ -35,6 +35,7 @@ import { GraphSelectorBuilder } from 'pages/Graph/GraphSelector';
 import { isMultiCluster } from '../../config';
 import { KialiIcon } from 'config/KialiIcon';
 import { kebabToggleStyle } from 'styles/DropdownStyles';
+import { ReactNode } from 'react';
```

Comment:
Why have you imported `react` library again? We already have `import * as React from 'react';` so you can import ReactNode as `React.ReactNode`.

Reply:
Removed